### DISCUSSION TITLE
Export Drawer UX

### DISCUSSION THREAD

**Discussion by connorwinston** [directus/directus#24916] [Feature Requests]
### Describe the Bug

The Export Drawer's UX in the Data Studio, does not match the functionality of the actual service. The Export Drawer uses the API's Export Query Parameter which converts a JSON API response to a JSON, CSV, XML, or YAML file. This means that the export can not have duplicate keys, the order of the keys is not preserved, only top-level fields of the item are columns, etc. The way this is currently designed does not reflect those requirements from the field/column selection interface.

### Directus Version

v10

**Comment by rijkvanzanten** [comment]
What's the proposed change? I don't know what the actionable item is here

**Comment by connorwinston** [comment]
The field/column selection interface needs to represent that ordering is not guaranteed (right now it lets you drag and drop and order the fields), that you can't have duplicate fields/columns, and that you can't have 2 different levels (`translations` and `translations.name` will only return `translations.name`).
